.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 # see Black section in README
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 21.5b1
+    rev: 23.3.0
```

Comment:
this will cause conflict since it's not the same verywhere

Reply:
I cannot run black without this because of click. Shouldn't we update black everywhere to fix this issue?

Reply:
do you have the version of click that is in the requirements-dev.txt?

We can change it everywhere if this version is compatible with python 3.8 but I think there was some settings change so it might need modification of a bunch of file